scripts/get_graphids.py

```diff
@@ -0,0 +1,160 @@
+#! /usr/env python3
+
+import argparse
+import aiohttp
+import asyncio
+import async_timeout
+import posixpath
+from taskcluster.async import Index
+from taskcluster.async import Queue
+
+
+HG_URL = 'https://hg.mozilla.org/'
+
+REPOS = ('releases/mozilla-beta', 'releases/mozilla-release',
+         'releases/mozilla-esr52', 'mozilla-central')
+
+""" http://mozilla-version-control-tools.readthedocs.io/en/latest/hgmo/pushlog.html?highlight=json-pushes#json-pushes-command """  # noqa
+JSON_PUSHES = 'json-pushes?changeset={rev}'
```

Comment:
I use `{repo}/json-pushes?changeset={revision}&tipsonly=1&version=2&full=1` . I think `version=2` is [recommended](http://mozilla-version-control-tools.readthedocs.io/en/latest/hgmo/pushlog.html?highlight=json-pushes#writing-agents-that-consume-pushlog-data) though that also discourages `full=1`. `tipsonly=1` is probably helpful in reducing output.

Reply:
I don't need full=1 for this, but tipsonly and version=2 is a good idea. (I made some minor changes to the code that uses this for version=2)